lib/kitchen/provisioner/chef_base.rb

```diff
@@ -47,18 +48,6 @@ class ChefBase < Base
         providers/**/* recipes/**/* resources/**/* templates/**/*
       ].join(",")
 
-      default_config :chef_metadata_url do |provisioner|
```

Comment:
Is the `chef_metadata_url` now handled entirely in mixlib-install?  I wouldn't think this would need to be configurable....